command/format.go

```diff
@@ -45,12 +45,19 @@ func outputFormatTable(ui cli.Ui, s *api.Secret, whitespace bool) int {
 	input := make([]string, 0, 5)
 	input = append(input, fmt.Sprintf("Key %s Value", config.Delim))
 
-	if s.LeaseID != "" && s.LeaseDuration > 0 {
-		input = append(input, fmt.Sprintf("lease_id %s %s", config.Delim, s.LeaseID))
-		input = append(input, fmt.Sprintf(
-			"lease_duration %s %d", config.Delim, s.LeaseDuration))
-		input = append(input, fmt.Sprintf(
-			"lease_renewable %s %s", config.Delim, strconv.FormatBool(s.Renewable)))
+	if s.LeaseDuration > 0 {
+		if s.LeaseID != "" {
+			input = append(input, fmt.Sprintf("lease_id %s %s", config.Delim, s.LeaseID))
+			input = append(input, fmt.Sprintf(
+				"lease_duration %s %d", config.Delim, s.LeaseDuration))
+		} else {
+			input = append(input, fmt.Sprintf(
+				"ttl_seconds %s %d", config.Delim, s.LeaseDuration))
```

Comment:
I feel like it should be the same `lease_duration` for consistency. Otherwise you need to special case higher level tooling as well.